x/dclupgrade/keeper/msg_server_propose_upgrade.go

```diff
@@ -73,5 +73,10 @@ func (k msgServer) ProposeUpgrade(goCtx context.Context, msg *types.MsgProposeUp
 		k.SetApprovedUpgrade(ctx, approvedUpgrage)
 	}
 
+	_, isFound = k.GetRejectedUpgrade(ctx, msg.Plan.Name)
+	if isFound {
+		k.RemoveRejectedUpgrade(ctx, msg.Plan.Name)
+	}
```

Comment:
Why can't we just remove it without checking?
Why should we remove it immediately when propose received?